static/src/js/hierarchy.js

```diff
@@ -8,7 +8,7 @@ class Hierarchy {
   }
 
   getAncestors(node) {
-    if (!this.ancestorMap.hasOwnProperty(node)) {
+    if (!Object.prototype.hasOwnProperty.call(this.ancestorMap, node)) {
```

Comment:
Yuck!

From the docs:

> You may want to turn this rule off if your code only touches objects with hardcoded keys, and you will never use an object that shadows an `Object.prototype` method or which does not inherit from `Object.prototype`.

Can we turn it off?

Reply:
Good find, strongly agree on the "yuck"!

Reply:
Removed in ff1a143